app/assets/javascripts/analytics.js

```diff
@@ -0,0 +1,10 @@
+Blacklight.onLoad(function() {
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-7219229-32', 'auto');
+  ga('send', 'pageview');
+  console.log('load')
```

Comment:
Bah, was a bit too quick on the merge button.  Did you mean to leave this `console.log` here?